karma.conf.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function(config) {
+    config.set({
+
+        basePath: '',
+        frameworks: ['browserify', 'jasmine'],
+        files: [
+            'bower_components/angular/angular.js',
+            'bower_components/angular-animate/angular-animate.js',
```

Comment:
having to specify all of these would be a duplication for later on..
how about we just require the following:
* public/js/vendor.js
* public/js/partials.js
* public/js/app.js
* test files
so that we leverage the previous bower task